src/SprykerEco/Client/ComputopShipment/QuoteShipmentExpander/QuoteShipmentExpanderInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Client\ComputopShipment\QuoteShipmentExpander;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+
+interface QuoteShipmentExpanderInterface
```

Comment:
```suggestion
interface QuoteDefaultShipmentExpanderInterface
```

Reply:
@gechetspr blocked such name and insisted on curent